<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Honeycomb edge</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="navContainer" native="true">
       <property name="minimumSize">
        <size>
         <width>260</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>260</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #FFFDF2; border-right: 1px solid #E0E0E0;</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item alignment="Qt::AlignLeft">
         <widget class="QLabel" name="logoLabel">
          <property name="minimumSize">
           <size>
            <width>220</width>
            <height>120</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="resources.qrc">:/resources/honeycomblogo.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="margin">
           <number>15</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="navPanel">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <item>
           <property name="text">
            <string>Dashboard</string>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
          </item>
          <item>
           <property name="text">
            <string>System Status</string>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Device Services</string>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Device Profiles</string>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Devices</string>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Events &amp; Readings</string>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Rules Engine</string>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Notifications</string>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnSettings">
          <property name="text">
           <string>Settings</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton { background-color: transparent; color: #555555; text-align: left; padding-left: 20px; border: none; font-size: 14px; font-weight: 500; height: 40px; margin: 5px; } QPushButton:hover { background-color: #FFF9D5; color: #F9942A; }</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QStackedWidget" name="centralStack"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
